.github/workflows/docs.yml

```diff
@@ -2,6 +2,8 @@ on:
   push:
     branches: [master]
   pull_request:
+  release:
```

Comment:
We don't really tag new releases so far, we just push stuff to the `crate` branch iirc. Not sure if that will change in the next release

Reply:
`gtk-rs/gtk-rs` has no branch crate, or do i miss something.
I simple assumed that there where no releases since the move from `gtk-rs/gtk` repo.

Reply:
Yes, because we still haven't done a release with the new repository structure. No idea if that will change in the future. But I am definitely in favour of tagging new releases :-)